x.payment_batching/payment_batching.md

```diff
@@ -0,0 +1,258 @@
+technique.
+
+### Delays
+
+This is the primary concern with payment batching.  Although some
+situations naturally lend themselves to payment batching (e.g. a mining
+pool paying hashrate providers in a block the pool mined), many
+services primarily send money to users when those users make a
+withdrawal request.  In order to batch payments, the service must get
+the user to accept that their payment will not be sent immediately---it
+will be held for some period of time and then combined with other
+withdrawal requests.
+
+Users will notice this delay because they won't receive a notification
+in their receiving wallet that an unconfirmed transaction is on the way
+until you send the batch containing their payment.  Also by delaying
+sending of their payment, you also delay when it's confirmed (all other
+things being equal, such as feerates).
+
+To mitigate the problem of delays, you may allow the
+user to choose between an immediate payment and a delayed payment with
+a different fee provided for each option.  For example:
+
+    [X] Free withdrawal (payment sent within 6 hours)
+    [ ] Immediate withdrawal (withdrawal fee 0.123 mBTC)
+
+### Reduced privacy
+
+A second concern with payment batching is that it can make users feel
+like they have less privacy.  Every user you pay in the same transaction
+can reasonably assume that everyone else receiving an output from that
+transaction is being paid by you.  If you had sent separate
+transactions, any onchain relationship between the payments might be
+less apparent or even non-existent.
+
+![Screenshot of a possible transaction batch in a block explorer](img/batch-screenshot.png)
+
+Note that transactions belonging to particular Bitcoin services are
+often identifiable by experts even if they don't use payment
+batching, so batching doesn't necessarily cause a reduction in privacy
+for those cases.
+
+It may be possible to partially mitigate this problem by sending batched
+payments in a coinjoin transaction created with other users.  Depending
+on the technique used, this would not necessarily reduce the efficiency
+of batching and could provide significantly enhanced privacy.  However,
+naive implementations of coinjoin previously provided by Bitcoin
+services have had [flaws][coinjoin sudoku] that prevented them from
+providing significant privacy advantages.  As of February 2019, no
+currently-available coinjoin implementation is fully compatible with the
+needs of payment batching.
+
+### Possible inability to fee bump
+
+A final concern is that you may not be able to fee bump a batched
+payment.  Transaction relay nodes such as Bitcoin Core impose limits on
+the transactions they relay to prevent attackers from wasting bandwidth,
+CPU, and other node resources.  By yourself, you can easily avoid
+reaching these limits, but the receivers of the payments you send can
+respend that money in child transactions that become part of the
```

Comment:
consider s/that money/the unspent outputs/ to emphasize that this is only a problem while the tx in unconfirmed.